meinberlin/apps/dashboard2/mixins.py

```diff
@@ -51,19 +52,8 @@ def blueprint_key(self):
         return self.kwargs['blueprint_slug']
 
 
-class DashboardComponentMixin(base.ContextMixin,
-                              base.View):
-
-    def dispatch(self, request, project, module, *args, **kwargs):
-        self.module = module
-        self.project = project
-        return super().dispatch(request, *args, **kwargs)
-
-    def get_context_data(self, **kwargs):
-        context = super().get_context_data(**kwargs)
-        context['project'] = self.project
-        context['module'] = self.module
-        return context
+class DashboardComponentMixin(ProjectContextDispatcher):
+    component = None
```

Comment:
I am not questioning whether these classes should inherit. I am just wandering about the names of these classes since the hierarchy is typically `Dashboard -> Project, Module`. Maybe the inheriting class should be called `ProjectDashboard*`.